test/Microsoft.AspNet.Mvc.Core.Test/Rendering/DefaultDisplayTemplatesTest.cs

```diff
@@ -71,17 +71,14 @@ public void ObjectTemplateDisplaysNullDisplayTextWhenObjectIsNull()
         {
             // Arrange
             var html = DefaultTemplatesUtilities.GetHtmlHelper();
-            var metadata =
-                new EmptyModelMetadataProvider()
-                    .GetMetadataForType(null, typeof(DefaultTemplatesUtilities.ObjectTemplateModel));
-            metadata.NullDisplayText = "(null value)";
-            html.ViewData.ModelMetadata = metadata;
```

Comment:
This is already an `HtmlHelper<DefaultTemplatesUtilities.ObjectTemplateModel>`. This is just removing redundant code.